charts/internal/machine-controller-manager/seed/templates/deployment.yaml

```diff
@@ -83,6 +81,35 @@ spec:
         - mountPath: /var/lib/machine-controller-manager
           name: machine-controller-manager
           readOnly: true
+      - name: provider-openstack
+        image: {{ index .Values.images "machine-controller-manager-provider-openstack" }}
+        imagePullPolicy: Always
```

Comment:
Why `Always` would be `IfNotPresent` also sufficient?

Reply:
Nice catch. It was a development artifact.